packages/amplify-category-storage/src/provider-utils/awscloudformation/cdk-stack-builder/s3-stack-transform.ts

```diff
@@ -333,7 +334,7 @@ export class AmplifyS3ResourceStackTransform {
   }
 
   //Helper: Save DependsOn entries to amplify-meta
-  _saveDependsOnToBackendConfig() {
+  _saveDependsOnToBackendConfig( ) {
```

Comment:
this could be simplified: 

```
const dependsOn = [...s3DependsOnResources || []];
```

and then just 1 call to `updateamplifyMetaAfterResourceUpdate`